src/main/java/com/revature/entities/Question.java

```diff
@@ -29,10 +29,10 @@
 
 	// add the not null check in the service layer
 	@Column(name = "creation_date")
-	private LocalDate creationDate;
```

Comment:
Why are we replacing LocalDate with Date?  Date is not recommended practice in Java - we should be using LocalDate or some other type from the `java.time` package when working with time.

Reply:
I was asked to change this by Chris because it was not recording the hours and seconds correctly in the database.

Reply:
Discussed in Slack, but as a note for anyone else who happens upon this convo, my recommendation would be to use LocalDateTime to include time tracking information.  While `java.util.Date` is not deprecated, I would not recommend it for new code due to a number of limitations regarding time zones, localization, and concurrency.

Reply:
Fixed